website/markdown/docs/usage/adoption-guidelines.mdx

```diff
@@ -0,0 +1,89 @@
+---
+name: Adoption guidelines
+excerpt: This page contains guidelines for migrating existing Xcode projects to Tuist.
+---
+
+# Adoption guidelines
+
+Unless you bootstrap a new project using Tuist,
+in which case you get everything configured automatically,
+you'll have to define your Xcode projects using Tuist's primitives _(e.g. Project.swift and Workspace.swift files)_.
+How tedious this process is,
+depends on how complex your projects are.
+
+As you probably know,
+Xcode doesn't prevent developers from adding accidental complexity.
+That leads to complexity in the shape of _groups that don't match the directory structure_,
+_files that are shared across targets_, or
+_file references that point to nonexisting files_ _(to mention some)._
+All that accumulated complexity makes it hard for us to provide a command that reliably ports migrates project.
+
+Moreover, manual migration is an **excellent exercise to clean up and simplify** your projects.
+Not only the developers in your project will be thankful for that,
+but Xcode,
+who will be faster processing and indexing them.
+Once you have fully adopted Tuist,
+it will make sure that projects are consistently defined and that they remain simple.
+
+In the aim of easing that work,
+we are giving you some guidelines based on the feedback that we have received from the users.
+
+<Message
+  info
+  title="Migration commands"
+  description="We are currently working on providing commands to assist you in the process. Until those commands are available, you might consider building simple CLI tools that leverage [XcodeProj](https://github.com/tuist/xcodeproj)"
+/>
+
+## Guidelines
+
+#### Extract your build settings into `.xcconfig` files
+
+The leaner the projects, the easier they are to migrate.
+To make them thinner,
+you can extract information like build settings that can be defined outside Xcode projects.
+You must prevent developers from defining settings using build settings.
+You can do that by implementing a script that reads the build settings using [XcodeProj](https://github.com/tuist/xcodeproj),
+and fails if the settings of projects and targets are not empty.
+
+#### Migrate the most dependant projects first
+
+Those are usually simpler and contain fewer dependencies than the rest.
```

Comment:
Do you want to mention `tuist graph` here as a command to use to see the dependency graph for a target? Also, we could easily build a script that outputs targets by dependencies

Reply:
It'd be great to have a similar command that works with Xcode projects. I doubt `tuist graph` is useful until you have fully migrated.